app/gui2/src/components/GraphEditor/GraphNode.vue

```diff
@@ -11,15 +11,35 @@ import { injectGraphSelection } from '@/providers/graphSelection'
 import { useGraphStore, type Node } from '@/stores/graph'
 import { useProjectStore } from '@/stores/project'
 import { useApproach } from '@/util/animation'
+import { AstExtended } from '@/util/ast'
+import { extractMatches } from '@/util/ast/match'
+import { escape as astStringEscape } from '@/util/ast/string'
 import { usePointer, useResizeObserver } from '@/util/events'
 import { displayedIconOf } from '@/util/getIconName'
 import type { Opt } from '@/util/opt'
 import { Rect } from '@/util/rect'
 import { Vec2 } from '@/util/vec2'
 import { setIfUndefined } from 'lib0/map'
-import type { ContentRange, ExprId, VisualizationIdentifier } from 'shared/yjsModel'
+import { type ContentRange, type ExprId, type VisualizationIdentifier } from 'shared/yjsModel'
 import { computed, ref, watch, watchEffect } from 'vue'
 
+const enableMethodName = 'with_enabled_context'
+const disableMethodName = 'with_disabled_context'
+function withOverriddenOutputContextPatternText(name: string, enable: boolean) {
```

Comment:
Do we need the `PatternText` suffix here? I think the function sounds ok without it.